pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -194,6 +194,36 @@ def dump_flash(
         else:
             return content
 
+    @EspSerial.use_esptool
+    def erase_partition(self, partition_name: str = None) -> None:
+        """
+        Erase the partition provided
+
+        Args:
+        partition_name: partition name
+        """
+        if partition_name is None:
+            raise ValueError('Invalid arguments')
+
```

Comment:
we could remove these two lines, would check here: `if partition_name in self.app.partition_table`

if you really want to check, I think

```suggestion
        if not partition_name:
            raise ValueError('Invalid arguments')
```

would be better, since it would also check if it's an empty string.

but overall, this check here is a bit redundant